src/components/com_tjcertificate/administrator/libraries/certificate.php

```diff
@@ -723,7 +705,13 @@ public function issueCertificate($replacements, $options)
 
 			if (!empty($this->expired_on))
 			{
-				if (!(DateTime::createFromFormat('Y-m-d H:i:s', $this->expired_on) !== false))
+				// Check if only date is provided e.g. Y-m-d, then add 23:59:59
+				if (!(DateTime::createFromFormat('Y-m-d H:i:s', $this->expired_on) !== false)
+					&& (DateTime::createFromFormat('Y-m-d', $this->expired_on) !== false))
+				{
```

Comment:
Consider changing this condition to `if ($this->expired_on != DateTime::createFromFormat('Y-m-d H:i:s', $this->expired_on))` ?